### PR TITLE
Android Engineer Wanted

### PR DESCRIPTION
**Pull request by rtsisyk**
###

This one a bit ChatGPT-ish. Let's polish it together.

### CODE REVIEW

**Review by oleg-rswll — APPROVED** [review]

File: content/contribute/android/_index.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+travelers around the world.
+
+### What You Will Do
+
+- Develop innovative consumer experiences on Android using Java and NDK.
+- Take ownership of features from inception to delivery, including technical debt
+  management and testing.
+- Set high standards for code quality, maintainability, and performance, while
+  mentoring other team members.
+- Handle complex features that span across mobile applications and our robust
```
Comment (pastk) [inline content/contribute/android/_index.md:40]: for sure it needs to be brought to OM's reality and needs :)

File: content/contribute/android/_index.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+### Opportunity
+
+Contribute to building a world-class navigation app that sets a new standard
+for privacy and offline functionality, all while remaining independent of corporate
+interests. You will play a crucial role in shaping the future of navigation for
+travelers around the world.
+
+### What You Will Do
+
+- Develop and implement Android app features through production.
```
Comment (oleg-rswll) [inline content/contribute/android/_index.md:37]: Remove first line

**Review by oleg-rswll — COMMENTED** [review]
- Develop and implement Android app features through production.
- Design technical solutions which operate effectively at scale.


- Collaborate with product, UI/UX design, engineering and support to create an intuitive and user-friendly experience.

- Partner with a global open-source community to drive development.


100% remote work from nearly anywhere in the world. Full time position. Part-time work or combining with other commitments is certainly possible.

**Comment by pastk** [comment]
I understand the new positions pages are not linked from anywhere on the web-site yet? E.g. https://organicmaps.app/contribute/android/ is accessibly directly via the url but there is no link from the Contribute page.

**Pull request merged**